students/amirg/lesson05/assignment/test_database.py

```diff
@@ -0,0 +1,69 @@
+"""
+Tests the database module
+"""
+from unittest import TestCase
+from database import import_data
+from database import show_available_products
+from database import show_rentals
+
+class DatabaseTests(TestCase):
+    """
+    Tests for the database module
+    """
+    def test_import_data(self):
+        """
+        Tests the import_data function
+        """
+        directory_path = r"C:/Users/Amir G/SP_Python220B_2019/students/amirg/" \
```

Comment:
Try use instead relative paths. Using hard-coded paths like in this case forces anyone trying to run your tests to hunt for these paths to replace them with the appropriate locations (that was my case).